src/style.css

```
@@ -1,3 +1,65 @@
 body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
```

```suggestion
  color: #000;
```

use hex color everywhere pls

---

-[DEMO LINK](https://Laisens.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Laisens.github.io/layout_moyo-header/report/html_report/)